evm.md

```diff
@@ -1372,12 +1345,38 @@ The various `CALL*` (and other inter-contract control flow) operations will be d
 
     syntax KItem ::= "#initVM"
  // --------------------------
-    rule <k> #initVM    => . ...      </k>
-         <pc>         _ => 0          </pc>
-         <memoryUsed> _ => 0          </memoryUsed>
-         <output>     _ => .ByteArray </output>
-         <wordStack>  _ => .WordStack </wordStack>
-         <localMem>   _ => .Map       </localMem>
+    rule <k> #initVM      => . ...      </k>
+         <pc>           _ => 0          </pc>
+         <memoryUsed>   _ => 0          </memoryUsed>
+         <output>       _ => .ByteArray </output>
+         <wordStack>    _ => .WordStack </wordStack>
+         <localMem>     _ => .Map       </localMem>
+         <jumpDests>    _ => #computeValidJumpDests(PGM) </jumpDests>
+         <program>      PGM             </program>
```

Comment:
It seems to me that we should actually do `#computeValidJumpDests` above where the `PGM` is actually loaded into the `<program>` cell, to ensure that the invariant that `<jumpDests>` cell contents always corresponds to `<program>`.

`initVM` should only be doing stuff like clearing state, not loading program (according to how it was written before).